src/openforms/registrations/tests/test_pre_registration.py

```diff
@@ -2,16 +2,30 @@
 
 from django.test import TestCase
 
+from rest_framework.exceptions import ValidationError
+
+from openforms.config.models import GlobalConfiguration
 from openforms.registrations.contrib.zgw_apis.tests.factories import (
     ZGWApiGroupConfigFactory,
 )
-from openforms.registrations.exceptions import RegistrationFailed
 from openforms.registrations.tasks import register_submission
+from openforms.submissions.constants import PostSubmissionEvents
 from openforms.submissions.tasks.registration import pre_registration
 from openforms.submissions.tests.factories import SubmissionFactory
 
 
 class PreRegistrationTests(TestCase):
+    def test_pre_registration_with_submission_not_completed(self):
+        submission = SubmissionFactory.create()
+
+        with self.assertLogs() as logs:
+            pre_registration(submission.id, PostSubmissionEvents.on_completion)
+
+        self.assertEqual(
+            logs.records[0].msg,
+            "Trying to pre-register submission '%s' which is not completed.",
+        )
```

Comment:
just for my understanding - is it deliberate that the placeholders are not interpolated?